### DISCUSSION TITLE
Access validators of pydantic as a normal function

### DISCUSSION THREAD

**Discussion by yeus** [pydantic/pydantic#2486] [Question] [answered]
Hi everyone,

is it possible to use pydantic validators as a "normal" function?

for example something like this:

    pydantic.AnyHttpUrl.validate(my_url)

but that doesn't work, as it needs a model in order to work...

I would be really happy to have this function as I could throw out a lot of other libraries that I use for validation in that case.

**Comment by ghost** [comment, marked as answer]
You can do
```python
from pydantic import parse_obj_as

parse_obj_as(AnyHttpUrl, 'http://www.example.com')
```

**Reply by yeus** [reply]
thx. Didn't figure that out from the [documentation](https://pydantic-docs.helpmanual.io/usage/models/#parsing-data-into-a-specified-type) because I thought it only worked on custom models.
